### ISSUE TITLE
[BUG] can't run tracee container

### ISSUE THREAD

**Issue by AsafEitani** [aquasecurity/tracee#1911]
## Prerequisites

- [ ] This affects latest released version.
- [x] This affects current development tree (origin/HEAD).
- [x] There isn't an issue describing the bug.

Select one OR another:

- [ ] I'm going to create a PR to solve this (assign to yourself).
- [x] Someone else should solve this.

## Bug description
trying to run tracee with the docker image fails
This is how I'm executing tracee (cmdline):
```
 sudo docker run \
>   --name tracee --rm -it \
>   --pid=host --cgroupns=host --privileged \
>   -v /etc/os-release:/etc/os-release-host:ro \
>   -e LIBBPFGO_OSRELEASE_FILE=/etc/os-release-host \
>   aquasec/tracee:latest
WARNING: The requested image's platform (linux/amd64) does not match the detecte                                d host platform (linux/arm64/v8) and no specific platform was requested
exec /tracee/entrypoint.sh: exec format error
```

## Context

Relevant information about my setup:

* Linux version: 20.04
* Linux kernel version: 5.13
* Tracee version (or commit id of your tree): latest
* LLVM version: none
* Golang version: none

**Comment by yanivagman** [comment]
Are you using arm64 arch? the image artifact is compiled for x86, and not for arm64

**Comment by AnaisUrlichs** [comment]
When I install the containers on Kubernetes through the Helm Chart from the Aqua reposiotry I get in the container logs:
> exec /tracee/entrypoint.sh: exec format error 

and the pod is in CrashLoopBackOff.
Followed the command in the installation: https://aquasecurity.github.io/tracee/dev/getting-started/installing/kubernetes/

@josedonizetti any ideas why that is? or how I can fix it?

**Comment by josedonizetti** [comment]
@AnaisUrlichs can you create a new issue for it, and provide more information please? what k8s are you using? how you installing it? Because I have just tested it on minikube and it worked.

**Comment by yanivagman** [comment]
As explained in #2757 , this is due to not having ARM64 docker images and will be resolved by #1913
